interface/aces.gfx

```diff
@@ -0,0 +1,322 @@
+		texturefile = "gfx/aces/FIN_ace_m_1.dds"
+	}
+	spriteType = {
+		name = "GFX_FIN_ace_m_2"
+		texturefile = "gfx/aces/FIN_ace_m_2.dds"
+	}
+	# FIN - AAT#
+	spriteType = {
+		name = "GFX_FIN_ace_Ilmari_Juutilainen"
+		texturefile = "gfx/aces/FIN_ilmari_juutilainen_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_FIN_ace_Jorma_Sarvanto"
+		texturefile = "gfx/aces/FIN_jorma_sarvanto_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_FIN_ace_Hans_Wind"
+		texturefile = "gfx/aces/FIN_hans_wind_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_FIN_ace_Oiva_Tuominen"
+		texturefile = "gfx/aces/FIN_oiva_tuominen_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_FIN_ace_Jorma_Karhunen"
+		texturefile = "gfx/aces/FIN_jorma_karhunen_ace.dds"
+	}
+
+	# DEN #
+	spriteType = {
+		name = "GFX_DEN_ace_m_0"
+		texturefile = "gfx/aces/DEN_ace_m_0.dds"
+	}
+	spriteType = {
+		name = "GFX_DEN_ace_m_1"
+		texturefile = "gfx/aces/western_european_ace_m_1.dds"
+	}
+	spriteType = {
+		name = "GFX_DEN_ace_m_2"
+		texturefile = "gfx/aces/western_european_ace_m_2.dds"
+	}
+
+	#spriteType = { # May do historical portraits if want to
+	#	name = "GFX_GER_ace_Erich_Hartmann"
+	#	texturefile = "gfx/aces/GER_ace_Erich_Hartmann.dds"
+	#}
+
+	# AST #
+	spriteType = {
+		name = "GFX_AST_ace_m_0"
+		texturefile = "gfx/aces/AST_axe_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_AST_ace_m_1"
+		texturefile = "gfx/aces/western_european_ace_m_1.dds"
+	}
+	spriteType = {
+		name = "GFX_AST_ace_m_2"
+		texturefile = "gfx/aces/western_european_ace_m_2.dds"
+	}
+
+	# USA #
+	spriteType = {
+		name = "GFX_USA_ace_Amelia_Earhart"
+		texturefile = "gfx/aces/Amelia_Earhart_ace.dds"
+	}
+
+	# SOV #
+	spriteType = {
+		name = "GFX_SOV_ace_Lidiya_Litvyak"
+		texturefile = "gfx/aces/SOV_lidiya_litvyak_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_SOV_ace_Yevdokiya_Bershanskaya"
+		texturefile = "gfx/aces/SOV_yevdokiya_bershanskaya_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_SOV_ace_Marina_Raskova"
+		texturefile = "gfx/aces/SOV_marina_raskova_ace.dds"
+	}
+
+	# ITA #
+	spriteType = {
+		name = "GFX_ITA_ace_Franco_Lucchini"
+		texturefile = "gfx/aces/ITA_franco_lucchini_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_ITA_ace_Carlo_Faggioni"
+		texturefile = "gfx/aces/ITA_carlo_faggioni_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_ITA_ace_Mario_Visintini"
+		texturefile = "gfx/aces/ITA_mario_visintini_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_ITA_ace_Adriano_Visconti"
+		texturefile = "gfx/aces/ITA_adriano_visconti_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_ITA_ace_Giulio_Graziani" #TODO_Manu: Real name is Giulio Cesare Graziani but portrait won't hook in with double name
+		texturefile = "gfx/aces/ITA_giulio_cesare_graziani_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_ITA_ace_Vittorio_Sanseverino"
+		texturefile = "gfx/aces/ITA_vittorio_sanseverino_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_ITA_ace_Giuseppe_Cenni"
+		texturefile = "gfx/aces/ITA_giuseppe_cenni_ace.dds"
+	}
+
+
+	### EXECUTRIX ###
```

Comment:
TODO - create texture files for imperial aces and modify these lines to include them